app/helpers/orders_helper.rb

```diff
@@ -1,2 +1,85 @@
 module OrdersHelper
+  def frendly_shipped_status(shipped_status)
+    shipped_status ? "Shipped" : "Not Shipped"
+  end
+
+  def pending_orders
+    orders = 0
+    @order_items.each do |order_item|
```

Comment:
I think this block could simplify to something like: `orders = @order_items.orders.where(status: 'pending').count`